db.go

```diff
@@ -0,0 +1,197 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"math"
+	"os"
+	"strings"
+	"time"
+	"unicode"
+
+	_ "github.com/mattn/go-sqlite3"
+)
+
+type saveDb struct {
+	db        *sql.DB
+	tableName string
+}
+
+const (
+	dbLocation              = "./tcping.db"
```

Comment:
we need to make this a mandatory input. For example, if the program is located in `/bin`  or similar locations, then `--db` will fail due to lack of privilege since it is not run as **root**.

the ideal way should be the user providing the path and the name of the DB like: `tcping example.com 443 --db /tmp/tcping.db`

This way, the program will not panic/crash due to permissions and the user is consciously dictates where the DB should be stored.

Reply:
hey @pouriyajamshidi 
https://github.com/pouriyajamshidi/tcping/blob/6a418489e474884f049f67d9d58e7cb88a0abc55/tcping.go#L319C1-L319C33
I think it is causing issue
When `--db file/path` is used then the value of the flag output becomes the `hostname` 
But `--db=file/path` then works fine.
Am I right?
Should I fix the `permuteArgs` func?

Reply:
ah, the `permuteArgs` is showing its age 😄 

I propose not fixing it in this PR. let's exclude the `db.go` from build by adding this line at the top of the file:

```go
//go:build exclude
```

once this merges in, if you want, you can open a separate PR to fix that function.

Reply:
ok 😄

Reply:
by the way, just to be clear, by not fixing it I mean the `permuteArgs` function not the DB path 😄

Reply:
hey @pouriyajamshidi 
you can use this library 
https://github.com/tdewolff/argp

Reply:
hey @wizsk,

Thanks for the suggestion. We would probably go with [cobra](https://github.com/spf13/cobra)